src/Components/Authorization/style.module.scss

```diff
@@ -0,0 +1,69 @@
+@import "../../styles/variabels.scss";
```

Comment:
file with module styles should be separated for every component. It is better to create a folder for each component and move `.tsx` and `.module.scss` inside